### DISCUSSION TITLE
3.7.1

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#787] [Announcements]
- 修复首次启动点击登录无效

----

### 下载

<details>
<summary>点击展开</summary>

[//]: # (注意, `checkLatestVersion` 有字符串处理, 修改标题和分隔符前务必查询)

[github-win-x64]: https://github.com/open-ani/ani/releases/download/v3.7.1/ani-3.7.1-windows-x86_64.zip

[github-mac-x64]: https://github.com/open-ani/ani/releases/download/v3.7.1/ani-3.7.1-macos-x86_64.dmg

[github-mac-aarch64]: https://github.com/open-ani/ani/releases/download/v3.7.1/ani-3.7.1-macos-aarch64.dmg

[github-android]: https://github.com/open-ani/ani/releases/download/v3.7.1/ani-3.7.1.apk

[cf-win-x64]: https://d.myani.org/v3.7.1/ani-3.7.1-windows-x86_64.zip

[cf-mac-x64]: https://d.myani.org/v3.7.1/ani-3.7.1-macos-x86_64.dmg

[cf-mac-aarch64]: https://d.myani.org/v3.7.1/ani-3.7.1-macos-aarch64.dmg

[cf-android]: https://d.myani.org/v3.7.1/ani-3.7.1.apk

[ghproxy-win-x64]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.7.1%2Fani-3.7.1-windows-x86_64.zip

[ghproxy-mac-x64]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.7.1%2Fani-3.7.1-macos-x86_64.dmg

[ghproxy-mac-aarch64]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.7.1%2Fani-3.7.1-macos-aarch64.dmg

[ghproxy-android]: https://mirror.ghproxy.com/?q=https%3A%2F%2Fgithub.com%2Fopen-ani%2Fani%2Freleases%2Fdownload%2Fv3.7.1%2Fani-3.7.1.apk

[qb-enhanced]: https://github.com/c0re100/qBittorrent-Enhanced-Edition/releases/latest

- PC 首次播放在线数据源时, 可能需要加载 10-30 秒。
- macOS 安装后如果无法打开: [解决方案](https://github.com/open-ani/ani/wiki/macOS-%E6%97%A0%E6%B3%95%E6%89%93%E5%BC%80%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88)
- Windows 打开时如果显示异常: [解决方案](https://github.com/open-ani/ani/wiki/Windows-%E4%B8%8B%E5%AD%97%E4%BD%93%E4%B8%8E%E8%83%8C%E6%99%AF%E9%A2%9C%E8%89%B2%E5%BC%82%E5%B8%B8)

| 操作系统                    | 全球                           | 中国大陆                                             |
|-------------------------|------------------------------|--------------------------------------------------|
| Windows x86_64          | [GitHub][github-win-x64]     | [主线][cf-win-x64] / [备线][ghproxy-win-x64]         |
| macOS x86_64 (Intel 芯片) | [GitHub][github-mac-x64]     | [主线][cf-mac-x64] / [备线][ghproxy-mac-x64]         |
| macOS aarch64 (M 系列芯片)  | [GitHub][github-mac-aarch64] | [主线][cf-mac-aarch64] / [备线][ghproxy-mac-aarch64] |
| Android APK aarch64     | [GitHub][github-android]     | [主线][cf-android] / [备线][ghproxy-android]         |

扫描二维码下载 Android 版本：

[github-android-qr]: https://github.com/open-ani/ani/releases/download/v3.7.1/ani-3.7.1.apk.github.qrcode.png

[cf-android-qr]: https://d.myani.org/v3.7.1/ani-3.7.1.apk.cloudflare.qrcode.png

| 全球                           | 中国大陆                         |
|------------------------------|------------------------------|
| ![GitHub][github-android-qr] | ![Cloudflare][cf-android-qr] |

</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/open-ani/ani/releases/tag/v3.7.1'>3.7.1</a>.</em>

**Comment by wuyuchengd** [comment]
反馈一个问题，就是搜番的时候，例如邪神与厨二病的第二季，会搜出来一个带逗号的结果，但在数据源查找的时候会查不到资源，如果单纯搜邪神与厨二病，在数据源里就会出现第二季
